security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,121 @@
+//OpenShift Compliance Operator Release Notes
+[id="compliance-operator-release-notes"]
+= Compliance Operator release notes
+:context: compliance-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+[id="compliance-operator-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright's message].
+
+[id="compliance-operator-release-notes-0-1-44"]
+== Release Notes for Compliance Operator 0.1.44
+Issued: 2021-
+
+The Compliance Operator 0.1.44 is now available with enhancements and bug fixes.
+
+[id="compliance-operator-0-1-44-new-features-and-enhancements"]
+=== New features and enhancements
+* Add option to make scan scheduling strict/not strict - adds a new ComplianceScan/Suite/ScanSetting option called strictNode scan. This option defaults to true, meaning that the operator will error out of a scan pod can't be scheduled. Switching the option to true makes the scan more permissive and go forward. Useful for clouds with ephemeral nodes.
++
+* Result Server: Make nodeSelector and tolerations configurable - exposes the nodeSelector and tolerations attributes through the ScanSettings object. This enables deployers to configure where the Result Server will run, and thus what node will host the Persistent Volume that will contain the raw results. This is needed for cases where the storage driver doesn't allow us to schedule a pod that makes use of a persistent volume on the master nodes.
+
+
+[id="openshift-compliance-operator-0-1-44-bug-fixes"]
+=== Bug fixes
+* Previously, a crash was occurring. In this release, the Compliance Operator now uses uses the latest openscap 1.3.5 to avoid this potential crash.
```

Comment:
Total nit, but could we say that previously a crash was occuring in one of the `scanner` containers of the pods performing the scan? Just to make it very clear that it wasn't CO itself that was crashing.

Reply:
Yes, thanks for the clarification. You might notice some other areas where I took a stab at rewriting, but need some additional info. All comments are welcome.